packages/headless/src/modules/brands/repositories/index.ts

```diff
@@ -0,0 +1,35 @@
+import { Product } from '../../../entity/product';
+import { IBrandDatabase } from './brand.database.interface';
+import { Injectable } from '@nestjs/common';
+import { Brand } from 'src/entity/brand';
+
+@Injectable()
+export class BrandRepository {
+    constructor(private readonly db: IBrandDatabase) { }
+
+    async findBrandById(brandId: string): Promise<Brand | null> {
+        const brand = await this.db.getBrandById(brandId);
+        return brand;
+    }
+
+    async findAllBrands(skip?: number, limit?: number): Promise<Brand[]> {
+        const brands = await this.db.getAllBrands(skip, limit);
```

Comment:
Instead of 2 lines of code, we can return it on the first line.
like `return await this.db.getAllBrands(skip, limit);`